### ISSUE TITLE
Lora in paperspace

### ISSUE THREAD

**Issue by legendan** [TheLastBen/fast-stable-diffusion#2117]
Good afternoon! Can you tell me if it is possible to upload a lora and use it in paperspace

**Comment by TheLastBen** [comment]
yes you can, upload it to the right folder just like you would upload any file

**Comment by iragonz** [comment]
> yes you can, upload it to the right folder just like you would upload any file

The problem with that is that it takes really long for long archives, is there a way to download them directly from drive/civitai/huggingface? Like how we can download a model from a link or the controlnet. Also store them on the temp storage.

**Comment by TheLastBen** [comment]
you can use `wget` to download from any link

**Comment by creadordigitalok** [comment]
> you can use `wget` to download from any link

TheLastBen, ¿automatic1111 notebook no longer works with Colab free? ¿Only in Colab Pro? Thanks!

**Comment by TheLastBen** [comment]
Colab is disconnecting free users from active sd sessions after 10 minutes

**Comment by creadordigitalok** [comment]
> Colab is disconnecting free users from active sd sessions after 10 minutes

And why do they do that? If they want to remove Colab Free, they shouldn't assign you any GPU directly. That's what I do not understand. And thanks for answering!

**Comment by TheLastBen** [comment]
they just don't want free users to use SD, they want them to pay

**Comment by iragonz** [comment]
> you can use `wget` to download from any link

Honestly, I don't even know how to access the code in Paperspace. I've found a workaround using your cell to download the model from a link to download loras, embeddings, etc and then renaming the archives and moving them to the correct place, it's ugly and a pain in the ass, but it works.

**Comment by creadordigitalok** [comment]
> they just don't want free users to use SD, they want them to pay

Thanks for the information

**Comment by cmatrix7777** [comment]
> > you can use `wget` to download from any link
> 
> Honestly, I don't even know how to access the code in Paperspace. I've found a workaround using your cell to download the model from a link to download loras, embeddings, etc and then renaming the archives and moving them to the correct place, it's ugly and a pain in the ass, but it works.

could you explain in more detail? a video maybe?
